docs/content/docs/operators/functions.md

```diff
@@ -0,0 +1,236 @@
+
+/** Simple program that converts vectors to double arrays. */
+public class VectorToArrayExample {
+    public static void main(String[] args) {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        StreamTableEnvironment tEnv = StreamTableEnvironment.create(env);
+
+        // Generates input vector data.
+        List<Vector> vectors =
+                Arrays.asList(
+                        Vectors.dense(0.0, 0.0),
+                        Vectors.sparse(2, new int[] {1}, new double[] {1.0}));
+        Table inputTable =
+                tEnv.fromDataStream(env.fromCollection(vectors, VectorTypeInfo.INSTANCE))
+                        .as("vector");
+
+        // Converts each vector to a double array.
+        Table outputTable = inputTable.select($("vector"), vectorToArray($("vector")).as("array"));
+
+        // Extracts and displays the results.
+        for (CloseableIterator<Row> it = outputTable.execute().collect(); it.hasNext(); ) {
+            Row row = it.next();
+            Vector vector = row.getFieldAs("vector");
+            Double[] doubleArray = row.getFieldAs("array");
+            System.out.printf(
+                    "Input vector: %s\tOutput double array: %s\n",
+                    vector, Arrays.toString(doubleArray));
+        }
+    }
+}
+```
+{{< /tab>}}
+
+{{< tab "Python">}}
+```python
+# Simple program that converts vectors to double arrays.
+
+from pyflink.common import Types
+from pyflink.datastream import StreamExecutionEnvironment
+from pyflink.table import StreamTableEnvironment
+
+from pyflink.ml.core.linalg import Vectors, VectorTypeInfo
+
+from pyflink.ml.lib.functions import vector_to_array
+from pyflink.table.expressions import col
+
+# create a new StreamExecutionEnvironment
+env = StreamExecutionEnvironment.get_execution_environment()
+
+# create a StreamTableEnvironment
+t_env = StreamTableEnvironment.create(env)
+
+# generate input vector data
+vectors = [
+    (Vectors.dense(0.0, 0.0),),
+    (Vectors.sparse(2, [1], [1.0]),),
+]
+input_table = t_env.from_data_stream(
+    env.from_collection(
+        vectors,
+        type_info=Types.ROW_NAMED(
+            ['vector'],
+            [VectorTypeInfo()])
+    ))
+
+# convert each vector to a double array
+output_table = input_table.select(vector_to_array(col('vector')).alias('array'))
+
+# extract and display the results
+output_value = [x for x in
+                t_env.to_data_stream(output_table).map(lambda r: r).execute_and_collect()]
+
+output_value.sort(key=lambda x: x[0])
+
+field_names = output_table.get_schema().get_field_names()
+for i in range(len(output_value)):
+    vector = vectors[i][0]
+    double_array = output_value[i][field_names.index("array")]
+    print("Input vector: %s \t output double array: %s" % (vector, double_array))
+
+```
+{{< /tab>}}
+
+{{< /tabs>}}
+
+### arrayToVector
+
+This function converts numerical arrays into vectors.
```

Comment:
Would it be useful to mention the specific python class of the output value like below? Same for related comments in this PR.

This function converts a column of arrays of numeric type into a column of pyflink.ml.core.linalg.DenseVector instances.